webapp/src/Controller/Jury/ConfigController.php

```diff
@@ -103,6 +103,10 @@ private function logUnverifiedJudgings(EventLogService $eventLogService)
     public function indexAction(EventLogService $eventLogService, Request $request)
     {
         $specs = $this->config->getConfigSpecification();
+        foreach ($specs as &$spec) {
```

Comment:
I have to admit, I have no ideas what you do in those four lines.

Reply:
Loading the $spec by reference, then adding options, which basically merges in the options on the fly.
The `unset` is needed because PHP will otherwise use the first time $spec is used after this as a pass-by-reference instead of pass-by-value and shit will hit all kinds of fans.